platform/ios/app_delegate.mm

```diff
@@ -92,8 +92,28 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 
 	mainViewController = viewController;
 
-	// prevent to stop music in another background app
-	[[AVAudioSession sharedInstance] setCategory:AVAudioSessionCategoryAmbient error:nil];
+	String categoryString = GLOBAL_GET("audio/general/ios/session_category");
+	AVAudioSessionCategory category;
```

Comment:
It's going to stay uninitialized if invalid category string is set (enum property type won't prevent it).